### PR TITLE
NEW Add option to hide customer code on PDF documents

### PR DESCRIPTION
**Pull request by BB2A-Anthony**
# New [*PDF not display customer code*]

### CODE REVIEW

**Comment by atm-maxime** [comment]
Usually, const names start with "MAIN_".
And if you add an option like this, you should apply it to all standard PDF documents, not just azure.
And I think there's a function "yn ()" that displays a yes/no select

**Comment by BB2A-Anthony** [comment]
> Usually, const names start with "MAIN_".
> And if you add an option like this, you should apply it to all standard PDF documents, not just azure.
> And I think there's a function "yn ()" that displays a yes/no select

Ok i have change const

You speek all standard PDF documents it's

- Project
- Propal
- Order
- Invoice
- Ticket
- Inter

Others?

I use yes / no as for the other options

File: htdocs/admin/pdf.php [hunk 1]
```diff
@@ -371,6 +373,18 @@
 //}
 print '</td></tr>';
 
+//
+
+print '<tr class="oddeven"><td>'.$langs->trans("DOC_SHOW_CUSTOMER_CODE");
```
Comment (eldy) [inline htdocs/admin/pdf.php:378]: Can you invert the option (the default value must be as actual, not set and showing the customer ref on document). And setting must hide.
So can you rename option 'DOC_SHOW_CUSTOMER_CODE' into 'MAIN_DOC_HIDE_CUSTOMER_CODE'

Comment (bb2a) [reply htdocs/admin/pdf.php]: It's ok for me

**Comment by BB2A-Anthony** [comment]
@eldy it's done

**Pull request merged**